ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.17.yaml

```diff
@@ -0,0 +1,357 @@
+    workflow: cucushift-installer-rehearse-azure-ipi-generation
+- as: azure-mag-ipi-fullyprivate-reboot-nodes-f14
+  cron: 25 6 4,18 * *
+  steps:
+    cluster_profile: azuremag-qe
+    env:
+      BASE_DOMAIN: qemag.azure.devcluster.openshift.com
+      ENABLE_REBOOT_CHECK: "true"
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-azure-mag-ipi-fullyprivate
+- as: azure-upi-custom-nsg-f28
+  cron: 34 0 27 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-azure-upi-custom-nsg
+- as: azure-upi-disconnected-f14
+  cron: 26 8 9,25 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-azure-upi-disconnected
+- as: azure-upi-proxy-reboot-nodes-f14
+  cron: 12 13 3,19 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      ENABLE_REBOOT_CHECK: "true"
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-azure-upi-proxy
+- as: gcp-ipi-cco-manual-users-static-f28
+  cron: 55 11 25 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-users-static
+- as: gcp-ipi-oidc-on-bastionhost-auth-with-sa-f28
+  cron: 37 9 5 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      EXTRACT_MANIFEST_INCLUDED: "true"
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-auth-with-sa
+- as: gcp-ipi-nested-virtualization-osdisk-type-size-f28
+  cron: 54 4 2 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-gcp-ipi-nested-virtualization-osdisk-type-size
+- as: gcp-ipi-xpn-cco-manual-users-static-f28
+  cron: 14 11 17 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-users-static
+- as: gcp-upi-xpn-f28
+  cron: 44 14 7 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-gcp-upi-xpn
+- as: nutanix-ipi-zones-single-osimage-customized-f28
+  cluster: build01
+  cron: 30 21 15 * *
+  steps:
+    cluster_profile: nutanix-qe-zone
+    env:
+      CLUSTER_OS_IMAGE: https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.17-9.4/builds/416.94.202405171948-0/x86_64/rhcos-416.94.202405171948-0-nutanix.x86_64.qcow2
```

Comment:
Seems this is an invalid address

Reply:
it's `https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.17-9.4/builds/417.94.202405291927-0/x86_64/rhcos-417.94.202405291927-0-nutanix.x86_64.qcow2` from https://github.com/openshift/installer/blob/release-4.17/data/data/coreos/rhcos.json#L618

Reply:
Good catch, it was changed by mistake when doing batch update.

Reply:
fixed it now.

Reply:
@sgaoshang do you want to use a 4.17 rhcos boot image now?

Reply:
okay, now I updated the boot image to the 4.17 one, should be harmless.